gov2/cloudfront/CreateDistribution/CreateDistribution.go

```diff
@@ -0,0 +1,134 @@
+
+	region := bucketResponseMetadata.Header.Get("x-amz-bucket-region")
+	originDomain := bucketName + ".s3." + region + ".amazonaws.com"
+
+	if err != nil {
+		return nil, err
+	}
+
+	cloudfrontResponse, err := cloudfrontClient.CreateDistribution(context.TODO(), &cloudfront.CreateDistributionInput{
+		DistributionConfig: &cloudfrontTypes.DistributionConfig{
+			Enabled:           aws.Bool(true),
+			CallerReference:   &originDomain,
+			Comment:           &originDomain,
+			IsIPV6Enabled:     aws.Bool(false),
+			PriceClass:        cloudfrontTypes.PriceClassPriceClass100,
+			HttpVersion:       cloudfrontTypes.HttpVersionHttp11,
+			DefaultRootObject: aws.String("index.html"),
+			Aliases: &cloudfrontTypes.Aliases{
+				Quantity: aws.Int32(1),
+				Items:    []string{domain},
+			},
+			ViewerCertificate: &cloudfrontTypes.ViewerCertificate{
+				ACMCertificateArn: aws.String(certificateSSLArn),
+				SSLSupportMethod:  cloudfrontTypes.SSLSupportMethodSniOnly,
+			},
+			CustomErrorResponses: &cloudfrontTypes.CustomErrorResponses{
+				Quantity: aws.Int32(1),
+				Items: []cloudfrontTypes.CustomErrorResponse{
+					{
+						ErrorCode:          aws.Int32(403),
+						ResponseCode:       aws.String("200"),
+						ErrorCachingMinTTL: aws.Int64(10),
+						ResponsePagePath:   aws.String("/index.html"),
+					},
+				},
+			},
+			Origins: &cloudfrontTypes.Origins{
+				Quantity: aws.Int32(1),
+				Items: []cloudfrontTypes.Origin{
+					{
+						DomainName: aws.String(originDomain),
+						Id:         aws.String(originDomain),
+						S3OriginConfig: &cloudfrontTypes.S3OriginConfig{
+							OriginAccessIdentity: aws.String("origin-access-identity/cloudfront/" + originAccessIdentityID),
+						},
+					},
+				},
+			},
+			CacheBehaviors: nil,
+			DefaultCacheBehavior: &cloudfrontTypes.DefaultCacheBehavior{
+				TargetOriginId:       aws.String(originDomain),
+				Compress:             aws.Bool(true),
+				MinTTL:               aws.Int64(200),
+				ViewerProtocolPolicy: cloudfrontTypes.ViewerProtocolPolicyRedirectToHttps,
+				ForwardedValues: &cloudfrontTypes.ForwardedValues{
+					Cookies: &cloudfrontTypes.CookiePreference{
+						Forward: cloudfrontTypes.ItemSelectionNone,
+					},
+					QueryString: aws.Bool(true),
+				},
+				AllowedMethods: &cloudfrontTypes.AllowedMethods{
+					Quantity: aws.Int32(2),
+					Items: []cloudfrontTypes.Method{
+						cloudfrontTypes.MethodGet,
+						cloudfrontTypes.MethodHead,
+					},
+				},
+			},
+		},
+	})
+
+	if err != nil {
+		return nil, err
+	}
+
+	return cloudfrontResponse, nil
+}
+
+func main() {
+	sdkConfig, err := config.LoadDefaultConfig(context.TODO())
+
+	if err != nil {
+		fmt.Println("Couldn't load default configuration. Have you set up your AWS account?")
+		fmt.Println(err)
+		return
+	}
+
+	s3Client := s3.NewFromConfig(sdkConfig)
+	cloudfrontClient := cloudfront.NewFromConfig(sdkConfig)
+	bucketName := "example-bucket-name"
+	originAccessIdentityID := "E12345678" // example id
```

Comment:
It's not clear how to get this value. Either include it in the code or explain how to get it.